Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/test/BlueprintBaseName.1.Tests/FunctionTest.cs

```diff
@@ -0,0 +1,32 @@
+using Xunit;
+using Amazon.Lambda.TestUtilities;
+using Moq;
+
+
+namespace BlueprintBaseName._1.Tests
+{
+    public class FunctionTest
```

Comment:
Why was the renamed from the pluaral `FunctionsTest`? Should we keep it pluaral to match the `Functions` class in the project?

Reply:
I don't remember either, might have been when rebasing to catch up from last year? Added back as plural.